tests/system/CommonFunctionsTest.php

```diff
@@ -391,7 +391,7 @@ public function testReallyWritable()
     public function testSlashItem()
     {
         $this->assertSame('/', slash_item('cookiePath')); // slash already there
-        $this->assertSame('', slash_item('cookieDomain')); // empty, so untouched
+        $this->assertNull(null, slash_item('cookieDomain')); // empty, so untouched
```

Comment:
The `$cookieDomain` is empty string. The changed assertion is incorrect.
https://github.com/codeigniter4/CodeIgniter4/blob/58aa05ee8a02994bc28e631cfe5aa3130831c259/app/Config/App.php#L261

Reply:
`public static function assertNull($actual, string $message = ''): void`

`$this->assertNull(null)` does not make sense.